### ISSUE TITLE
Resolution plot

### ISSUE THREAD

**Issue by alexkjames** [LinkedEarth/Pyleoclim_util#357]
**Is your feature request related to a problem? Please describe.**
It would be nice to have a way to automatically plot the resolution of a time series within pyleoclim. It can be useful when exploring new records for the first time and for a sanity check after applying interpolation, binning, etc. 

**Describe the solution you'd like**
A Series based plotting method that fits into our ecosystem that plots the resolution of the time series. Could call it `res_plot`, though that might be confusing with the existence of stuff like `ssa_res`, so open to other names. Maybe `resolution_plot`?

**Comment by CommonClimate** [comment]
Using [tsbase.resolution](https://github.com/LinkedEarth/Pyleoclim_util/blob/c3bb80df4257242b490bbf3acf3f0947a6eda112/pyleoclim/utils/tsbase.py#L419), create new Resolution class in a module resolution.py . 

Add it to init.py

Populate with at least 3 methods:

- describe() (inspired by pandas)
- plot()
- histplot() (see NB1 of Pyleoclim paper) 
```
dt=np.diff(ts_benthic.time) # calculate the difference between adjacent time
fig, ax = plt.subplots(figsize=[10,4])
sns.histplot(dt,kde=True,ax=ax) 
ax.set_xlabel('Age increments (ka)')
```

**Comment by alexkjames** [comment]
closed with #394
